src/main/java/com/stripe/param/SubscriptionCreateParams.java

```diff
@@ -0,0 +1,758 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class SubscriptionCreateParams extends ApiRequestParams {
+  /**
+   * A non-negative decimal between 0 and 100, with at most two decimal places. This represents the
+   * percentage of the subscription invoice subtotal that will be transferred to the application
+   * owner's Stripe account. The request must be made with an OAuth key in order to set an
+   * application fee percentage. For more information, see the application fees
+   * [documentation](https://stripe.com/docs/connect/subscriptions#collecting-fees-on-subscriptions).
+   */
+  @SerializedName("application_fee_percent")
+  BigDecimal applicationFeePercent;
+
+  /**
+   * Either `charge_automatically`, or `send_invoice`. When charging automatically, Stripe will
+   * attempt to pay this subscription at the end of the cycle using the default source attached to
+   * the customer. When sending an invoice, Stripe will email your customer an invoice with payment
+   * instructions. Defaults to `charge_automatically`.
+   */
+  @SerializedName("billing")
+  Billing billing;
+
+  /**
+   * A future timestamp to anchor the subscription's [billing
+   * cycle](https://stripe.com/docs/subscriptions/billing-cycle). This is used to determine the date
+   * of the first full invoice, and, for plans with `month` or `year` intervals, the day of the
+   * month for subsequent invoices.
+   */
+  @SerializedName("billing_cycle_anchor")
+  String billingCycleAnchor;
```

Comment:
This should be `long` right? since it's a unix timestamp

Reply:
@remi-stripe, I realized I haven't fixed this type inconsistencies